python-spec/src/somacore/spatial.py

```diff
@@ -685,54 +673,13 @@ def level_count(self) -> int:
         raise NotImplementedError()
 
     @abc.abstractmethod
-    def level_properties(self, level: Union[int, str]) -> "ImageProperties":
+    def level_shape(self, level: Union[int, str]) -> Tuple[int, ...]:
         """The properties of an image at the specified level.
 
         Lifecycle: experimental
         """
         raise NotImplementedError()
 
-    @property
-    def reference_level(self) -> Optional[int]:
-        """The index of image level that is used as a reference level.
-
-        This will return ``None`` if no current image level matches the size of the
-        reference level.
-
-        Lifecycle: experimental
-        """
-        raise NotImplementedError()
-
-    @property
-    @abc.abstractmethod
-    def reference_level_properties(self) -> "ImageProperties":
-        """The image properties of the reference level.
-
-        Lifecycle: experimental
-        """
-        raise NotImplementedError()
-
-
-class ImageProperties(Protocol):
-    """Class requirements for level properties of images.
-
-    Lifecycle: experimental
-    """
-
-    @property
-    def name(self) -> str:
-        """The key for the image.
-
-        Lifecycle: experimental
-        """
-
-    @property
-    def shape(self) -> Tuple[int, ...]:
-        """Size of each axis of the image.
-
-        Lifecycle: experimental
-        """
```

Comment:
This class didn't add much value as is. We may want to expand it to include more information or just remove it altogether.